pkgs/development/python-modules/ezdxf/default.nix

```diff
@@ -21,6 +21,11 @@ buildPythonPackage rec {
     sha256 = "sha256-x1p9dWrbDtDreXdBuzOA4Za+ZC40y4xdEU7MGb9uUec=";
   };
 
+  postPatch = ''
+    substituteInPlace tests/test_02_dxf_graphics/test_215_dimension.py \
+      --replace "dim.dxf.angle == 45" "math.isclose(dim.dxf.angle, 45)"
+  '';
```

Comment:
Let's not do that kind of patching downstream. Instead report the problem upstream and disable the test for the time being.

Reply:
Done. Also mozman/ezdxf#516